src/vault.ts

```diff
@@ -5,7 +5,9 @@ const pexec = util.promisify(oexec);
 
 const slice = process.argv[2] == '.' ? 3 : 2
 const ARGS = process.argv.slice(slice);
-const OPTIONS = require('simple-argv')
+const yargs = require('yargs/yargs')
+const { hideBin } = require('yargs/helpers')
```

Comment:
@cfloress Could it be standardized to have the constants names in the same way (common convention is uppercase)

Reply:
Hi @marcoguillenuio , Jossemar's answer applies --> https://github.com/workflows-sh/aws-eks-ec2-asg-cdk/pull/16#discussion_r1213265368 , thanks @jossemargt-cto-ai